cardano/base/views.py

```diff
@@ -7,17 +7,27 @@ def index(request):
     countries = Country.objects.order_by('name', '-created').distinct('name')
     mozillians = Mozillian.objects.latest()
 
-    previous_week = Mozillian.objects.order_by('-created')[1]
```

Comment:
Instead of try/except you could check the count and accordingly select the correct element.